src/components/Global/Filter/index.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from 'react';
+import RadioButton from '../RadioButton';
+import styles from './Filter.module.scss';
+
+const Index = () => {
+  const [easyChecked, setEasyChecked] = useState(false);
+  const [mediumChecked, setMediumChecked] = useState(false);
+  const [hardChecked, setHardChecked] = useState(false);
+
```

Comment:
Just a suggestion
How about having an array of 3 and you will update the element inside that array
```js
const [levelChecker, setLevelChecker] = useState<Array<Boolean>>([false, false, false]);
```
and update will be
```js
const temp = [...levelChecker];
temp[1] = true;
setLevelChecker(temp);
```

Reply:
not sure if it will run faster but it will have less state to manage

Reply:
it's recommended to separate the states, so I'll leave them the way they are.
https://reactjs.org/docs/hooks-faq.html#should-i-use-one-or-many-state-variables